scripts/ci_ecs_benchmarks.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+# assign local ip
+ip=`curl http://169.254.169.254/latest/meta-data/local-ipv4`
+webserver_url=http://"$ip":9000
+da_webserver_url=http://"$ip":9001
+orchestrator_url=http://"$ip":4444
+
+# start webserver
+just async_std example webserver -- http://0.0.0.0:9000 &
+just async_std example webserver -- http://0.0.0.0:9001 &
+sleep 10m
```

Comment:
Are we sleeping 10m after we start the webservers? Do the webservers take long to start up? I don't think the validators start up until the orchestrator is ready after the sleep

Reply:
Oh I set this previously for CI, since it re-compile all the things for these two commands. But I should re-think about this. Let me remove this ci script first.